src/main/java/com/niyati/designpattern/behavioral/chainOfResponsibility/AddNumbers.java

```diff
@@ -0,0 +1,19 @@
+package com.niyati.designpattern.behavioral.chainOfResponsibility;
+
```

Comment:
rename the package name, it should be all on lower case, also keep the package name short. instead chainOfResponsibility keep it like cor